drivers/modem/alt1250/altcom_lwm2m_hdlr.c

```diff
@@ -352,23 +352,89 @@ static int32_t fwup_srvop_handle(FAR uint8_t *pktbuf, size_t pktsz,
 }
 
 /****************************************************************************
- * name: fwupdate_notice_hndl
+ * name: parse_inst_number
  ****************************************************************************/
 
-static int32_t fwupdate_notice_hndl(FAR uint8_t *pktbuf, size_t pktsz,
-                          FAR void **cb_args, size_t arglen)
+static int parse_inst_number(FAR uint8_t **buf, FAR size_t *bufsz)
 {
-  return fwup_srvop_handle(pktbuf, pktsz, cb_args, arglen);
+  int ret = 0;
+
+  if (!isdigit(**buf))
+    {
+      return -1;
+    }
+
+  while (*bufsz)
+    {
+      if (!isdigit(**buf))
+        {
+          break;
+        }
+
+      ret = ret * 10 + ((**buf) - '0');
+      (*bufsz)--;
+      (*buf)++;
+    }
+
+  return ret;
 }
 
 /****************************************************************************
  * name: server_op_notice_hndl
  ****************************************************************************/
 
 static int32_t server_op_notice_hndl(FAR uint8_t *pktbuf, size_t pktsz,
-                          FAR void **cb_args, size_t arglen)
+                                     FAR void **cb_args, size_t arglen)
 {
-  return fwup_srvop_handle(pktbuf, pktsz, cb_args, arglen);
+  int i;
+  FAR int *event = (FAR int *)&cb_args[0];
+  FAR int *srvid = (FAR int *)&cb_args[1];
+  FAR int *inst  = (FAR int *)cb_args[2];
+
+  /* The valiable of "inst" is a type of struct lwm2mstub_instance_s in fact.
```

Comment:
```suggestion
  /* The valiable of "inst" is a type of struct lwm2mstub_instance_s in fact.
```

I think 'valiable' -- did you mean 'value' or 'variable' instead?